{
  "test_specification": {
    "encoding": "utf8_string",
    "log": "2022-11-09T10:27:00.570949+05:30 ulaa-u20-aio sudo: pam_unix(sudo:auth): authentication failure; logname=test uid=1001 euid=0 tty=/dev/pts/4 ruser=test rhost=  user=test"
  },
  "assertions": [
    {
      "assertion_type": "only_if_path_exists",
      "json_path": "original_string",
      "expected_pattern": "authentication",
      "negated_pattern": false,
      "description": "authentication",
      "active": true
    }
  ],
  "author": "siembol",
  "version": 2,
  "test_case_name": "authentication_failure_log",
  "config_name": "linux_syslog"
}